clustered/ehcache-clustered/src/assemble/docker/server/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM azul/zulu-openjdk-alpine:17-jre-latest
+
+RUN apk update && apk upgrade && apk add bash
+RUN mkdir /terracotta && chgrp -R 0 /terracotta && chmod -R g+rX /terracotta
+RUN mkdir /terracotta/config && chgrp -R 0 /terracotta/config && chmod -R g+rX /terracotta/config
+RUN mkdir /terracotta/run && chgrp -R 0 /terracotta/run && chmod -R g+rwX /terracotta/run
+
+WORKDIR /terracotta
+
+# This volume can be mounted with "ro" flag.
+# This is where the configurations will be sourced and eventually copied elsewhere within the container.
+VOLUME "/terracotta/config"
+
+# This folder can be mounted with "rw" flag.
+# This is where the default configurations will output any logs, data, etc
+# If the volume is mounted, the container will support being restarted in the same state.
+# If a volume is mounted, be sure to set the correct chmod permissions in the host machine.
+# Also, you might need to use a special cleanup task to remove the created files in the host machine
+VOLUME "/terracotta/run"
+
+EXPOSE 9410
+
+ENV DEFAULT_OFFHEAP "offheap-1:512MB,offheap-2:512MB"
+ENV DEFAULT_ACTIVATE "false"
```

Comment:
Would it make sense for this to be true for OSS images?

Reply:
Good question... It would help users I think but... The problem is that this won't be consistent with the use of the docker image with a custom CLI (not through using env vars by tweaking the default CLI). When using CLI, the auto activate flag has to be added to auto activate.